myria/cluster/scripts/cli.py

```diff
@@ -1141,7 +1141,20 @@ def run():
     help="Number of Myria workers per cluster node [default: %d]" % INSTANCE_TYPE_CONFIGS[DEFAULTS['instance_type']].workers_per_node)
 @click.option('--cluster-log-level', show_default=True,
     type=click.Choice(LOG_LEVELS), default=DEFAULTS['cluster_log_level'])
+@click.option('--perfenforce', is_flag=True, help="Enable PerfEnforce (will override default cluster configuration)")
 def create_cluster(cluster_name, **kwargs):
+    # If perfenforce is enabled, override the cluster configuration
+    if kwargs['perfenforce']:
+        click.echo("Adjusting cluster options for PerfEnforce...")
```

Comment:
What is an easy way to check for arguments specified by users? For example, currently I have a callback that checks if `ctx.params.get('instance_type')` has a value assigned. Even if the user does not specify `instance-type`, this parameter is already given a default value.  How can I check for the actual arguments given?